### ISSUE TITLE
Error when executing `mdsine2.negbin.build_graph()`

### ISSUE THREAD

**Issue by alejandrocs98** [gerberlab/MDSINE2#18]
I get this error when executing `mdsine2.negbin.build_graph()` (I get this error even when using the _Gibson_ dataset loaded using `mdsine2.dataset.load_gibson()`):
```
[DEBUG] Available subjects: 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15
[DEBUG] Subject 1, available qpcr: 1.0,2.0,3.0,5.0,7.0,10.0,12.0,13.0,15.0,16.0,17.0,19.0,21.0,25.0,26.0,27.0,29.0,30.0,31.0,33.0,35.0,38.0,40.0,41.0,42.0
[DEBUG] Subject 2, available qpcr: 1.0,2.0,3.0,5.0,7.0,10.0,12.0,13.0,15.0,16.0,17.0,19.0,21.0,25.0,26.0,27.0,29.0,30.0,31.0,33.0,35.0,38.0,40.0,41.0,42.0
[DEBUG] Subject 3, available qpcr: 1.0,2.0,3.0,5.0,7.0,10.0,12.0,13.0,15.0,16.0,17.0,19.0,21.0,25.0,26.0,27.0,29.0,30.0,31.0,33.0,35.0,38.0,40.0,41.0,42.0

[CRITICAL] Failed in Filtering
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/tmp/ipykernel_416345/3628062348.py in <module>
      2     params=params,
      3     graph_name=mcnulty.name,
----> 4     subjset=mcnulty)

~/anaconda3/envs/mdsine2/lib/python3.7/site-packages/mdsine2/negbin.py in build_graph(params, graph_name, subjset)
   1016     for name in params.INITIALIZATION_ORDER:
   1017         try:
-> 1018             GRAPH[name].initialize(**params.INITIALIZATION_KWARGS[name])
   1019         except:
   1020             logger.critical('Failed in {}'.format(name))

~/anaconda3/envs/mdsine2/lib/python3.7/site-packages/mdsine2/negbin.py in initialize(self, tune, end_tune, target_acceptance_rate, qpcr_variance_inflation, delay)
    558         # Initialize the trajectory sets
    559         for ridx in range(self.G.data.n_replicates):
--> 560             self.value[ridx].initialize()
    561 
    562         if self.mp == 'full':

~/anaconda3/envs/mdsine2/lib/python3.7/site-packages/mdsine2/negbin.py in initialize(self)
    432         rel = np.sum(self.data, axis=1)
    433         rel = rel / np.sum(rel)
--> 434         value = rel * self.qpcr_measurement.mean()
    435 
    436         self.value = np.zeros(len(value))
AttributeError: 'qpcr_measurement' object has no attribute 'mean'
```

**Comment by yk23** [comment]
What happens when you use the command line interface instead? `mdsine2 infer-negbin -i <path_to_study_pkl>`?

This might actually be related to your other issue  #17

**Comment by yk23** [comment]
Confirmed to be fixed in development branch `cv-wip-mirror`.

**Comment by yk23** [comment]
Addressed in #19
